model.go

```diff
@@ -18,23 +18,24 @@ type ExecCommand interface {
 type (
 	// Generic Worker config
 	Config struct {
-		AccessToken                string `json:"accessToken"`
-		ClientId                   string `json:"clientId"`
-		LiveLogCertificate         string `json:"livelogCertificate"`
-		LiveLogExecutable          string `json:"livelogExecutable"`
-		LiveLogKey                 string `json:"livelogKey"`
-		LiveLogSecret              string `json:"livelogSecret"`
-		Certificate                string `json:"certificate"`
-		ProvisionerId              string `json:"provisionerId"`
-		RefreshUrlsPrematurelySecs int    `json:"refreshURLsPrematurelySecs"`
-		PublicIP                   net.IP `json:"publicIP"`
-		Subdomain                  string `json:"subdomain"`
-		WorkerGroup                string `json:"workerGroup"`
-		WorkerId                   string `json:"workerId"`
-		WorkerType                 string `json:"workerType"`
-		UsersDir                   string `json:"usersDir"`
-		CleanUpTaskDirs            bool   `json:"cleanUpTaskDirs"`
-		IdleShutdownTimeoutSecs    int    `json:"idleShutdownTimeoutSecs"`
+		AccessToken                string                 `json:"accessToken"`
```

Comment:
Yes, that happens automatically because `go fmt` lines up the columns, and there is a new entry added with a type that is longer than the previous types were (`map[string]interface{}`).